### PR TITLE
Add Getting Started guide, adds Minishift tutorial

### PR DESCRIPTION
**Pull request by cdrage**
Updates the getting started guide as well as introduces an
"introduction" page for Kompose.

Adds a Minishift guide as well as a general update to all documentation.

### CODE REVIEW

**Comment by cdrage** [comment]
@pradeepto Both CDK/RHEL guide as well as Minishift (upstream) guide has now been added. Giving us a total of three getting started guides: minikube + kompose, minishift + kompose, rhel (cdk) + kompose.

File: docs/getting-started.md [hunk 1]
```diff
@@ -0,0 +1,347 @@
...
+## Minishift and Kompose
+
+In this guide, we'll deploy a sample `docker-compose.yaml` file to an OpenShift cluster.
+
+Requirements:
+  - [minishift](https://github.com/minishift/minishift)
+  - [kompose](https://github.com/kubernetes/kompose)
+  - An OpenShift route created
+
+__Note:__ The service will NOT be accessible until you create an OpenShift route with `oc expose`. You must also have a virtualization environment setup. By default, `minishift` uses VirtualBox.
```
Comment (surajnarwade) [inline docs/getting-started.md]: By default, `minishift` uses KVM in linux, Hyer-V in windows and xhyve in OS X

Reference: https://www.openshift.org/minishift/

File: docs/getting-started.md [hunk 2]
```diff
@@ -0,0 +1,347 @@
...
+
+__Start `minishift`:__
+
+[Minishift](https://github.com/minishift/minishift) is a tool that helps run OpenShift locally using a single-node cluster inside of a VM. Similar to [minikube](https://github.com/kubernetes/minikube].
```
Comment (surajnarwade) [inline docs/getting-started.md]: Broken hyperlink:
it should be
[minikube](https://github.com/kubernetes/minikube)

File: docs/getting-started.md [hunk 3]
```diff
@@ -0,0 +1,347 @@
...
+INFO Successfully created ImageStream: redis-slave 
+
+Your application has been deployed to OpenShift. You can run 'oc get dc,svc,is,pvc' for details.
+```
+
+__Access the newly deployed service:__
+
+After deployment, you must create an OpenShift route in order to access the service.
+
+If you're using `minikube`, you'll use a combination of `oc` and `minishift` commands to access the service.
```
Comment (surajnarwade) [inline docs/getting-started.md]: replace `minikube` with `minishift`

Comment (surajnarwade) [reply docs/getting-started.md]: @cdrage ^^

Comment (cdrage) [reply docs/getting-started.md]: @surajnarwade GitHub is messing up. Ctrl+F5 and refresh the page and you should see that I've already replaced it.

**Comment by surajnarwade** [comment]
@cdrage you rock when it's doc thing :+1:

**Pull request merged**